### DISCUSSION TITLE
Why did you remove bulk edit of Sync Mode selection?

### DISCUSSION THREAD

**Discussion by OrangeManLives** [airbytehq/airbyte#32725] [Connector Questions]
Hello.

I recently attempted an upgrade of Airbyte from Version 0.40.28 to 0.50.34, hoping this would resolve a number of issues I've been experiencing with Airbyte.

After tackling the changes in upgrade method between these versions, we've discovered that the bulk edit of sync modes has since been removed.

The bulk edit was removed here: https://github.com/airbytehq/airbyte/issues/26080.
I believe because it was broken.

This functionality is important to our use, we have around 30 DBs/Connections with around 600 tables each.

Manually setting the sync mode per table is going to take forever.

Another user has reported this here; https://github.com/airbytehq/airbyte/discussions/30783

I've done some scripting to setup sources, destinations and connections via the Airbyte API, however I have avoided using the Airbyte API for Refreshing the source schema and attempting to enable streams/tables as this required the handling of many payloads, and it was easier to just get this done in less than 10 clicks in the interface.

Is there any chance the bulk edit of sync mode will make a return?
Could you let me know the airbyte version prior to this change?

**Comment by marcosmarxm** [comment]
The bulk edit was removed after some analysis it wasn't frequently used and when used caused confusion to users.
@OrangeManLives I'll keep the discussion open to get feedback from other users (upvote the discussion if you think the bulk editing is important). The current way to edit a bulk connection is using the Airbyte API.

**Comment by emilmar** [comment]
Thanks for providing a suggestion on how to bulk edit the sync mode. Perhaps that could be added to the documentation of sync modes, alternatively directly in the info box. I'd like for this feature to return! But yes, so far only 1 more person have liked this issue so I guess we're in a minority. We have 3 schemas with around 500 tables each (sharded table) which we need to change the sync mode of so I felt a bit crazy when I told our development team this feature _used_ to exist and they insisted I was hallucinating :)

[a comment by an automated account is not shown]
